lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -18,6 +19,20 @@ function distribute(node) {
   if (Node.Type.isUnaryMinus(node)) {
     return distributeUnaryMinus(node);
   }
+  else if (checks.canDistribute(node)) {
+    const [base, exponent] = node.args;
+
+    let newNode;
+    const times = parseFloat(exponent.value);
+
+    if (times > 0) {
+      newNode = Node.Creator.operator('*', Array(times).fill(base));
+      return distributeAndSimplifyOperationNode(newNode);
+    }
+    else {
```

Comment:
Like before, if you return in the `if` statement, you don't need to have an `else` statement.